addons/a11y/src/components/Report/Info.tsx

```diff
@@ -3,11 +3,10 @@ import React, { FunctionComponent } from 'react';
 import { styled } from '@storybook/theming';
 import { Result } from 'axe-core';
 
-const Wrapper = styled.div(({ theme }) => ({
-  backgroundColor: theme.background.bar,
```

Comment:
I think we should be updating `theme.background.bar` in this fix instead. As we want it to be responsive to theme changes

Reply:
Yes, i thought about this, but a11y addon not have a bg on other panels.

Reply:
indeed!

Reply:
Looking at it, I do think there's both a bug in the application of the correct color to theme.background.bar AND this element should have the background color removed!